IrisSoftware/main.py

```diff
@@ -39,28 +39,39 @@ def __init__(self) -> None:
 
         self.processingThread: threading.Thread
 
+        self.interpolator = Interpolator()
+        self.isCalibrated = False
```

Comment:
```suggestion
```
This should be removed. Instead of using `self.isCalibrated`, please use `self.state.isCalibrated`.